mysql-test/suite/component_js_lang/t/js_lang_big.test

```diff
@@ -0,0 +1,97 @@
+#
+# JS language component tests which require lot of memory or CPU.
+#
```

Comment:
The test is named "_big" but there is no `--source include/big_test.inc`

Reply:
Fixed.